src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -137,12 +137,6 @@ def test_gen_tool_by_source_error(self, tool_source, tool_type, error_code, erro
             gen_tool_by_source("fake_name", tool_source, tool_type, working_dir),
         assert str(ex.value) == error_message
 
-    def test(self):
```

Comment:
why remove this test?

Reply:
Note the right place and good name to have this test. Moved to test_utils since this test purpose is to test refresh_connections_dir. For collect_package_tools_and_connections, it is tested below in this test file.